.github/pull_request_template.md

```diff
@@ -26,6 +27,14 @@ Read the guidelines [here](https://github.com/civiform/civiform/wiki/Database#wr
 - [ ] Assigned two reviewers
 - [ ] Guarded against already existing resources using `IF NOT EXISTS` and `IF EXISTS`
 - [ ] Downs created to undo changes in Ups
+- [ ] Data migrations aren't being done (if so, a [Durable Job]((https://github.com/civiform/civiform/wiki/Database#durable-jobs-for-data-updates) should likely be used)
```

Comment:
suggestion: this language is a little confusing, at first read I thought it was saying Durable Jobs should be used if data migrations aren't being done. What do you think of this:
`Data migrations aren't being done (please use a [Durable Job](link) if doing a data migration`

Reply:
Updated!